### DISCUSSION TITLE
能否获取每日营收

### DISCUSSION THREAD

**Discussion by 15098731779** [Chia-Network/chia-blockchain#6457] [Support]
我们能不能通过api获取全网每日营收

**Comment by 15098731779** [comment]
我们能不能通过api获取全网每日营收

**Comment by 21497724** [comment]
https://api2.chiaexplorer.com/blockchainSummary

**Comment by BrandtH22** [comment]
@15098731779 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
